plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerLabelProvider.java

```diff
@@ -142,8 +146,6 @@ public StyledString getStyledText(Object element) {
 			case ResourceKind.SERVICE:
 				IService service = (IService) resource;
 				return getStyledText(service);
-			case ResourceKind.TEMPLATE:
```

Comment:
mmkay, but I still strongly believe that non-used code is a liability. I dont see a need (for now) that justify this to remain in.